app/controllers/deliveries/new/select-recipient.js

```diff
@@ -0,0 +1,49 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  queryParams: ['projectId'],
+  projectId: null,
+  project: Ember.computed('projectId', function () {
+    return this.get('store').findRecord('duke-ds-project', this.get('projectId'));
+  }),
+  application: Ember.inject.controller(),
+  currentDukeDsUser: Ember.computed.alias('application.currentDukeDsUser'),
+  validUsersList: Ember.computed('model.[]', function () {
+    // remove users with invalid fullName values
+    return this.get('model')
```

Comment:
I see users in my listing with empty email addresses too. Won't this be problematic too?

Reply:
![empty-emails](https://user-images.githubusercontent.com/2752599/43975198-d08b1468-9caa-11e8-84ea-c018bde574dc.png)

Reply:
I added code to filter out users without emails.